packages/odyssey-storybook/src/components/_internal/ThemeTable.tsx

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
In the past I have put these kinds of internal components in `packages/odyssey-storybook/.storybook/components`

I would like for us to stay consistent, but have no preference as to where these things should live. Will defer to you.

Reply:
Moved in 3edfdbe8
Feels better to have them out of src